Assets/Scripts/DefinitelyNotABait.cs

```diff
@@ -0,0 +1,114 @@
+using System.Collections;
+using System.Collections.Generic;
+using Unity.VisualScripting;
+using UnityEngine;
+
+public class DefinitelyNotABait : MonoBehaviour
+{
+    public GameObject blob;
+    
+    void Start()
+    {
+        StartCoroutine(Spawn());
+    }
+
+    
+    // Update is called once per frame
+    void Update()
+    {
+        VeryNecessaryCalculation(Random.Range(-1000, 10000), Random.Range(-900, 100020));
+    }
+    
+    
+    
+    IEnumerator Spawn()
+    {
+        while (true)
+        {
+            GameObject e = (GameObject)Instantiate(blob, new Vector3(Random.Range(0, 10), Random.Range(0, 10), 0), Quaternion.identity);
```

Comment:
Redundant cast to GameObject, since `Instantiate()` already returns a GameObject